main/integration-tests/helpers/api-param-generator.js

```diff
@@ -0,0 +1,345 @@
+}
+
+function deleteProjectParams(id) {
+  return { api: `${apiEndpoint}/api/projects/${id}` };
+}
+
+function addProjectParams(project) {
+  return { api: `${apiEndpoint}/api/projects`, body: project };
+}
+
+function updateProjectParams(project) {
+  return { api: `${apiEndpoint}/api/projects/${project.id}`, body: project };
+}
+
+function getAccountsParams() {
+  return { api: `${apiEndpoint}/api/accounts` };
+}
+
+function getAccountParams(id) {
+  return { api: `${apiEndpoint}/api/accounts/${id}` };
+}
+
+function removeAccountInfoParams(id) {
+  return { api: `${apiEndpoint}/api/accounts/${id}` };
+}
+
+function getComputePlatformsParams() {
+  return { api: `${apiEndpoint}/api/compute/platforms` };
+}
+
+function getComputeConfigurationsParams(platformId) {
+  return { api: `${apiEndpoint}/api/compute/platforms/${platformId}/configurations` };
+}
+
+function getClientIpAddressParams() {
+  return { api: `${apiEndpoint}/api/ip` };
+}
+
+function getScEnvironmentCostParams(id, numberDaysInPast, groupByService = true, groupByUser = false) {
+  return {
+    api: `${apiEndpoint}/api/costs?scEnv=${id}&numberOfDaysInPast=${numberDaysInPast}&groupByService=${groupByService}&groupByUser=${groupByUser}`,
+  };
+}
+
+function getScEnvironmentsParams() {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/` };
+}
+
+function getScEnvironmentParams(id) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${id}` };
+}
+
+function createScEnvironmentParams(scEnvironment) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/`, body: scEnvironment };
+}
+
+function createScEnvironmentConnectionUrlParams(envId, connectionId) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${envId}/connections/${connectionId}/url` };
+}
+
+function deleteScEnvironmentParams(id) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${id}` };
+}
+
+function stopScEnvironmentParams(id) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${id}/stop` };
+}
+
+function startScEnvironmentParams(id) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${id}/start` };
+}
+
+function updateScEnvironmentCidrsParams(id, updateRequest) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${id}/cidr`, body: updateRequest };
+}
+
+function getScEnvironmentConnectionsParams(envId) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${envId}/connections/` };
+}
+
+function sendSshKeyParams(envId, connectionId, keyPairId) {
+  return {
+    api: `${apiEndpoint}/api/workspaces/service-catalog/${envId}/connections/${connectionId}/send-ssh-public-key`,
+
+    body: { keyPairId },
+  };
+}
+
+function getWindowsRpInfoParams(envId, connectionId) {
+  return { api: `${apiEndpoint}/api/workspaces/service-catalog/${envId}/connections/${connectionId}/windows-rdp-info` };
+}
+
+module.exports = {
+  getAuthIdTokenParams,
+  addIndexParams,
+  listUsersParams,
+  addUsersParams,
```

Comment:
Two more user API in this controller - https://github.com/awslabs/service-workbench-on-aws/blob/master/addons/addon-base-rest-api/packages/base-controllers/lib/user-controller.js

Reply:
Added. Thanks